src/EditorFeatures/CSharpTest/CodeActions/EncapsulateField/EncapsulateFieldTests.cs

```diff
@@ -1255,5 +1255,89 @@ public int I
 }
 ", options: Option(SimplificationOptions.QualifyFieldAccess, true));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.EncapsulateField)]
+        public async Task Tuple()
+        {
+            var text = @"
+class foo
+{
+    private (int, string) b[|a|]r;
+
+    void baz()
+    {
+        var q = bar;
```

Comment:
As @jaredpar mentioned on some other PR, words like foo, bar, ... are getting flagged by a polycheck tool. It is better to not use them at all rather than confirming that context is appropriate for all occurrences.